code/modules/admin/verbs/adminpm.dm

```diff
@@ -261,7 +261,7 @@
 /datum/pm_tracker
 	var/current_title = ""
 	var/open = FALSE
-	var/list/pms = list()
+	var/list/datum/pm_convo/pms = list()
```

Comment:
But why?

Reply:
Lines such as `if(!pms[title.key].client)` utilise variables (client) which are tacked onto a `/datum/pm_convo`

Reply:
While typed lists may be meaningless normally, dreamchecker uses them for certain forms of validation